modules/Link.js

```diff
@@ -126,6 +126,11 @@ const Link = React.createClass({
             props.style = { ...props.style, ...activeStyle }
         }
       }
+
+      if (activeTitle) {
+        if (router.isActive(loc, onlyActiveOnIndex))
```

Comment:
I am not really happy with this check copied from line 121...
It should probably be stored in a variable but I don't know if this call is costly.